16-slices/exercises/16-internals-backing-array-fix/main.go

```diff
@@ -21,7 +21,7 @@ import (
 //
 // CURRENT OUTPUT (INCORRECT)
 //
-//  Mine         : [-50 -100 -150 25 30 50]
```

Comment:
@firasuke This was correct but this was incorrect, could you adjust this commit:
**This:**
```fmt.Println("Mine         :", mine[:3])```
**Should be:**
```fmt.Println("Mine         :", mine)```

Reply:
Done!